lib/pure/math.nim

```diff
@@ -67,6 +67,11 @@ when defined(c) or defined(cpp):
 
   proc c_signbit(x: SomeFloat): cint {.importc: "signbit", header: "<math.h>".}
 
+  func c_frexp(x: cfloat, exponent: var cint): cfloat {.
+      importc: "frexpf", header: "<math.h>".}
+  func c_frexp(x: cdouble, exponent: var cint): cdouble {.
```

Comment:
optional: add a vmops for frexp?

Reply:
I could do this in the future.

How I can register vmops for `func frexp*[T: float32|float64](x: T, exponent: var int): T =`?

Reply:
could be done in the future pr